tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/update_fields_numbers_as_strings_test.py

```diff
@@ -0,0 +1,128 @@
+"""
+Integration test for update_fields_numbers_as_strings cleaning rule
+
+Original Issues: DC-1052
+
+Background
+It has been discovered that the field type for some PPI survey answers is incorrect: there are several instances of
+numeric answers being saved as ‘string’ field types. The expected long-term fix is for PTSC to correct the field type
+on their end; however, there is no anticipated timeline for the completion of this work. As a result, the Curation team
+will need to create a cleaning rule to correct these errors.
+
+Cleaning rule to fill null values in value_as_number with values in value_as_string,
+EXCEPT when it’s a ‘PMI Skip’ for each of the observation_source_value
+
+Rule should be applied to the RDR export
+"""
+
+# Python Imports
+import os
+
+# Project Imports
+from app_identity import PROJECT_ID
+from cdr_cleaner.cleaning_rules.update_fields_numbers_as_strings import UpdateFieldsNumbersAsStrings
+from tests.integration_tests.data_steward.cdr_cleaner.cleaning_rules.bigquery_tests_base import BaseTest
+
+
+class UpdateFieldsNumbersAsStringsTest(BaseTest.CleaningRulesTestBase):
+
+    @classmethod
+    def setUpClass(cls):
+        print('**************************************************************')
+        print(cls.__name__)
+        print('**************************************************************')
+
+        super().initialize_class_vars()
+
+        cls.insert_fake_participant_tmpls = [
+            cls.jinja_env.from_string("""
+            INSERT INTO
+            `{{fq_table_name}}` 
+            (observation_id,
+            person_id,
+            observation_concept_id,
+            observation_date,
+            observation_type_concept_id,
+            value_as_number,
+            value_as_string, 
+            observation_source_value)
+            VALUES
+            (12345, 1111, 8621, DATE('2018-09-20'), 45905771, NULL, '29', 'ipaq_3_cope_a_24'),
+            (123456, 2222, 715713, DATE('2019-09-20'), 45905771, NULL, '1', 'lifestyle_2_xx12_cope_a_198'),
+            (1234567, 3333, 715723, DATE('2020-01-20'), 45905771, NULL, '10', 'lifestyle_2_xx12_cope_a_152'),
+            (12345678, 4444, 1333015, DATE('2017-11-20'), 45905771, 30, 'test', 'basics_xx'),
+            (123456789, 5555, 1333118, DATE('2015-10-13'), 45905771, NULL, '40', 'FAKE'),
+            (123456781, 6666, 1333118, DATE('2015-10-13'), 45905771, NULL, 'PMI Skip', 'cdc_covid_19_n_a2')
```

Comment:
Did you chose this as a subset of `OBSERVATION_SOURCE_VALUES`?

Reply:
I took the list that Francis provided in the ticket of `observation_source_value`'s to populate the fake data. I queried an observation table with those source value's to get the corresponding `observation_type_concept_id` and `observation_concept_id`'s